ci/Dockerfile

```diff
@@ -105,11 +88,14 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 # Install linuxdeployqt
 RUN mkdir -p /usr/local/bin \
- && curl -Lo/usr/local/bin/linuxdeployqt "https://status-misc.ams3.digitaloceanspaces.com/desktop/linuxdeployqt-${LINUXDEPLOYQT_VERSION}-x86_64.AppImage" \
+ && curl -Lo/usr/local/bin/linuxdeployqt "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage" \
```

Comment:
I actually don't remember why we had our own build.

Reply:
probably because you wanted to pin things.
This continuous deployment app image is built regularly and as time passes we may find ourseleves in a case where `linuxdeployqt` AppImage is too new for our setup.

Reply:
Yes, I think you're right. They just have nightly builds.

Reply:
I'll upload to digital ocean and use it.